packages/cozy-client/src/models/account.js

```diff
@@ -39,7 +36,9 @@ const DEFAULT_CONTRACT_SYNC_STATUS = true
 /**
  * Returns whether a contract is synced from account relationship
  *
- * @param  {CozyAccount} account - Cozy account
+ * @param  {import('../types').IOCozyAccount} account - Cozy account
+ * @param  {String} contractId - contract identifier
+ * @returns  {String} synchronisation status
```

Comment:
is it really a `string`? It seems that it can return at least a `true` boolean

Reply:
My mistake, in every case, this is a boolean